kolibri/plugins/coach/assets/src/views/groups-page/index.vue

```diff
@@ -79,16 +79,16 @@
     },
     computed: {
       showCreateGroupModal() {
-        return this.modalShown === GroupModals.CREATE_GROUP;
+        return this.groupModalShown === GroupModals.CREATE_GROUP;
```

Comment:
Not sure how I feel about this pattern. Couldn't we just have a method call an action to change the state? 

Also not seeing the point of having an object represent the modals. We can't display more than one modal at a time, so every modal that isn't active would be false?

I don't think you made these decisions, but I'd like your opinion